client/packages/system/src/Patient/PatientView/PatientView.tsx

```diff
@@ -98,6 +103,12 @@ const PatientDetailView = ({
     createNewPatient,
     setCreateNewPatient,
   } = usePatientStore();
+
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
```

Comment:
We have a custom hook `useUrlQuery` that we use for this (as mentioned above also).